### PR TITLE
RHINENG-13972 suggestions engine server setup

### PR DESCRIPTION
**Pull request by r14chandra**
## PR Title :boom:

Server setup for Suggestions Engine (renamed from PCP Generator).

## Why do we need this change? :thought_balloon:

Please include the __context of this change__ here.

## Documentation update? :memo:

- [ ] Yes
- [X] No

## Security Checklist :lock:

Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)

## :guardsman: Checklist :dart:

- [ ] Bugfix
- [X] New Feature
- [ ] Refactor
- [ ] Unittests Added
- [ ] DRY code
- [ ] Dependency Added
- [ ] DB Migration Added

## Additional :mega:

Feel free to add any other relevant details such as __links, notes, screenshots__, here.

### CODE REVIEW

File: ros/lib/config.py [hunk 1]
```diff
@@ -141,6 +141,7 @@ def build_endpoint_url(ep):
 DAYS_UNTIL_STALE = int(os.getenv("DAYS_UNTIL_STALE", '45'))
 CW_LOGGING_FORMAT = '%(asctime)s - %(levelname)s  - %(funcName)s - %(message)s'
 ROS_PROCESSOR_PORT = int(os.getenv("ROS_PROCESSOR_PORT", "8000"))
+ROS_SUGGESTIONS_ENGINE_PORT = int(os.getenv("ROS_SUGGESTIONS_ENGINE_PORT", "8000"))
```
Comment (upadhyeammit) [inline ros/lib/config.py:144]: hmm, I dont remember if we decided to share the 8000 as port or use new one?

**Review by upadhyeammit — COMMENTED** [review]
Looks good to me, I just dont remember about the port.

File: ros/lib/config.py (hunk 1 above)
Comment (r14chandra) [reply ros/lib/config.py:144]: We have set the same default port for both the processor and the API, so we will be doing the same for this service as well. However, when running locally, you must have all services running on different ports.

**Comment by r14chandra** [comment]
Merging the PR, @upadhyeammit. Thanks for the review!

**Pull request merged**